### DISCUSSION TITLE
Discussions!

### DISCUSSION THREAD

**Discussion by ToastCoder** [Gear-AI/Gear-AI.github.io#4] [General]
Let's discuss here and see if it is actually good

**Comment by a-imran99** [comment]
Can we have an elaboration of what is gear AI, any idea of design, workflow? or is it like a kind of python library for AI/ML with documentation.
@smv1999 @ToastCoder

**Reply by a-imran99** [reply]
would like to contribute :) !

**Reply by ToastCoder** [reply]
Oh, we are just a community that develops software but will mainly focus on AI Projects. We have started just now, so we're working on ideas. Our python library is a part of our planned projects.We will go a little bit slow since we're all busy with our academics as well. Sure, we would love to have your contributions in your particular domain. We are currently working on establishing everything. We'll let you know once we're ready in accepting public contributions. @a-imran99

**Reply by a-imran99** [reply]
Oh okay.
